### PR TITLE
Demo and code example

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-lookup?_sp.spa=pr-demo-and-code-example](https://host.nxt.blackbaud.com/skyux-lookup?_sp.spa=pr-demo-and-code-example)

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-lookup?_sp.spa=pr-demo-and-code-example](https://host.nxt.blackbaud.com/skyux-lookup?_sp.spa=pr-demo-and-code-example)

**Comment by Blackbaud-AdamFunderburk** [comment]
@Blackbaud-ToddRoberts Do we want to treat Disabled as an "option" on a component that requires a behavior demo?

I believe we made the hypothesis before that that Disabled was better represented as a "state", that we would document in the States table with just an image. Think there's a reason to make that an interactive part of the demo?

**Comment by Blackbaud-AdamFunderburk** [comment]
@Blackbaud-TrevorBurch What do you think about removing "Disabled" as an option from the behavior demo? I think we'd like our strategy to be using the States table elsewhere in the documentation to illustrate it.

**Comment by Blackbaud-TrevorBurch** [comment]
@Blackbaud-AdamFunderburk I'm fine with that. I just added it to add it. I'm definitely not tied down to it.

**Comment by Blackbaud-TrevorBurch** [comment]
@Blackbaud-AdamFunderburk I've removed that toggle. The PR link should update here in a few minutes with the demo without it.

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-lookup?_sp.spa=pr-demo-and-code-example](https://host.nxt.blackbaud.com/skyux-lookup?_sp.spa=pr-demo-and-code-example)

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-lookup?_sp.spa=pr-demo-and-code-example](https://host.nxt.blackbaud.com/skyux-lookup?_sp.spa=pr-demo-and-code-example)

File: src/app/public/plugin-resources/code-examples/country-field/basic/country-field-demo.component.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  selector: 'app-country-field-demo',
+  templateUrl: './country-field-demo.component.html'
+})
+export class CountryFieldDemoComponent implements OnInit {
+
+  public countryData: SkyCountryFieldCountry;
+
+  public countryForm: FormGroup;
+
+  public countryControl: FormControl;
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/plugin-resources/code-examples/country-field/basic/country-field-demo.component.ts]: Check your alphabetical ordering of properties here ☝️

File: src/app/docs/country-field/country-field-docs.component.ts [hunk 2]
```diff
@@ -1,11 +1,43 @@
...
   changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class CountryFieldDocsComponent {}
+export class CountryFieldDocsComponent implements OnInit {
+
+  public countryData: SkyCountryFieldCountry;
+
+  public countryForm: FormGroup;
+
+  public countryControl: FormControl;
```
Comment (Blackbaud-AlexKingman) [inline src/app/docs/country-field/country-field-docs.component.ts]: Check your alphabetical ordering of properties here ☝️

File: src/app/public/plugin-resources/code-examples/country-field/basic/country-field-demo.component.ts (hunk 1 above)
Comment (Blackbaud-TrevorBurch) [reply src/app/public/plugin-resources/code-examples/country-field/basic/country-field-demo.component.ts]: Done

File: src/app/docs/country-field/country-field-docs.component.ts (hunk 2 above)
Comment (Blackbaud-TrevorBurch) [reply src/app/docs/country-field/country-field-docs.component.ts]: Done

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-lookup?_sp.spa=pr-demo-and-code-example](https://host.nxt.blackbaud.com/skyux-lookup?_sp.spa=pr-demo-and-code-example)

**Comment by Blackbaud-TrevorBurch** [comment]
@Blackbaud-AlexKingman I addressed your ordering comments. As for your comment about the dropdown. We have an open `docs-tools` issue to address that https://github.com/blackbaud/skyux-docs-tools/issues/31

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-lookup?_sp.spa=pr-demo-and-code-example](https://host.nxt.blackbaud.com/skyux-lookup?_sp.spa=pr-demo-and-code-example)

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-lookup?_sp.spa=pr-demo-and-code-example](https://host.nxt.blackbaud.com/skyux-lookup?_sp.spa=pr-demo-and-code-example)

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-lookup?_sp.spa=pr-demo-and-code-example](https://host.nxt.blackbaud.com/skyux-lookup?_sp.spa=pr-demo-and-code-example)

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-lookup?_sp.spa=pr-demo-and-code-example](https://host.nxt.blackbaud.com/skyux-lookup?_sp.spa=pr-demo-and-code-example)

**Comment by Blackbaud-AdamFunderburk** [comment]
LGTM

**Pull request merged**